### ISSUE TITLE
bug(hubble): TypeScript type error with clearInterval in Node.js environment

### ISSUE THREAD

**Issue by hiroyukikumazawa** [farcasterxyz/hub-monorepo#2099]
**What is the bug?**
during build, typeScript raises the following type error:
```
No overload matches this call.
Overload 1 of 2, '(intervalId: string | number | Timeout | undefined): void', gave the following error:
Argument of type 'Timer | undefined' is not assignable to parameter of type 'string | number | Timeout | undefined'.
Type 'Timer' is not assignable to type 'string | number | Timeout | undefined'.
Property '[Symbol.dispose]' is missing in type 'Timer' but required in type 'Timeout'.
Overload 2 of 2, '(id: number | undefined): void', gave the following error:
Argument of type 'Timer | undefined' is not assignable to parameter of type 'number | undefined'.
Type 'Timer' is not assignable to type 'number'.ts(2769)
```

**Comment by sds** [comment]
Thanks for the report! Addressed in #2110.

**Comment by hiroyukikumazawa** [comment]
> Thanks for the report! Addressed in #2110.

welcome
